packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/notifservicepushapi.md

```diff
@@ -0,0 +1,60 @@
+Use this API to provide push provider credentials from FCM so that Okta can send push notifications to your custom app authenticator. See [Push Providers API](/docs/reference/api/push-providers/).
+
+<ApiOperation method="post" url="/api/v1/push-providers" />
```

Comment:
same comment as above on using this format outside of the API Reference.

```suggestion
<ApiOperation method="post" url="/api/v1/push-providers" />
```

Reply:
Ditto